src/containers/explore/components/EntityDataRow.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * @flow
+ */
+
+import React from 'react';
+
+import styled from 'styled-components';
+import { Cell, Colors, StyleUtils } from 'lattice-ui-kit';
+import { DataUtils } from 'lattice-utils';
+import { useDispatch } from 'react-redux';
+import type { UUID } from 'lattice';
+
+import { Routes } from '../../../core/router';
+import { goToRoute } from '../../../core/router/actions';
+
+const { NEUTRAL } = Colors;
+const { getEntityKeyId } = DataUtils;
+
+const StyledTableRow = styled.tr`
+  background-color: transparent;
+  border-bottom: 1px solid ${NEUTRAL.N100};
+  ${(props) => !props.isModal && StyleUtils.getHoverStyles};
```

Comment:
```suggestion
  ${(props) => !props.isModal && StyleUtils.getHoverStyles};
  
  &:last-child {
    border-bottom: none;
  }
```

... to get rid of that extra 1px border

<img width="1387" alt="Screen Shot 2021-06-10 at 8 45 50 PM" src="https://user-images.githubusercontent.com/440299/121627783-d96b6d00-ca2c-11eb-9759-20f5eab30e8a.png">